CONTRIBUTING.md

```diff
@@ -1,10 +1,14 @@
 # Contributing guidelines
 
-We gladly accept bugfixes and new gateways. Please follow the guidelines here to ensure your work is accepted.
+We gladly accept bugfixes, but are not actively looking to add new gateways. Please follow the guidelines here to ensure your work is accepted.
 
 ## New Gateways
 
-Please see the [ActiveMerchant Guide to Contributing a new Gateway](https://github.com/activemerchant/active_merchant/wiki/contributing) for information on adding a new gateway to ActiveMerchant.
+We're not taking on many new gateways at the moment. The team maintining ActiveMerchant is small and with the limited resources available, we generally prefer not to support a gateway than support a gateway poorly.
```

Comment:
Typo here. `maintining`

Reply:
Probably should also be "...prefer not to support a gateway than *to* support a gateway poorly"; it reads a bit awkwardly the way I wrote it originally.